ci-operator/step-registry/assisted/common/setup/prepare/assisted-common-setup-prepare-commands.sh

```diff
@@ -89,6 +89,7 @@ export TEST_TEARDOWN=false
 export TEST_FUNC=test_install
 export ASSISTED_SERVICE_HOST={{ IP }}
 export PUBLIC_CONTAINER_REGISTRIES="{{ CI_REGISTRIES | join(',') }}"
+export OPENSHIFT_INSTALL_RELEASE_IMAGE={{ OPENSHIFT_INSTALL_RELEASE_IMAGE }}
```

Comment:
so what will happen on the e2e-metal-assisted-4-y jobs?
Doesn't test-infra uses ``OPENSHIFT_INSTALL_RELEASE_IMAGE`` (if it presents) to create an AI instance that only supports one OCP version?

Reply:
We don't change the logic here as this variable was always set for periodics. The `e2e-metal-assisted-4-y` are using a variable `OPENSHIFT_VERSION`, I guess it overrides `OPENSHIFT_INSTALL_RELEASE_IMAGE` ?

Reply:
Yep, looks like the case: https://github.com/openshift/assisted-test-infra/blob/master/src/assisted_test_infra/test_infra/utils/utils.py#L466-L498

Reply:
Very well. Good to know we're doing the right thing in test-infra :)